attributes/default.rb

```diff
@@ -66,6 +66,10 @@
     '/etc/datadog-agent'
   end
 
+# The site of the Datadog intake to send Agent data to.
+# Defaults to 'datadoghq.com', set to 'datadoghq.eu' to send data to the EU site.
+default['datadog']['site'] = 'datadoghq.com'
```

Comment:
Ideally, we should leave the default to `nil` and only write `site` in the config if it's not `nil`.

This allows the Agent to define what the default should be, instead of the cookbook. This isn't the way most attributes were added to the cookbook (and changing that is a breaking change), but it allows greater flexibility and makes the cookbook easier to maintain.